protos/platform/v0/platform.proto

```diff
@@ -85,3 +86,15 @@ message GetIdentityIdsByPublicKeyHashesResponse {
   repeated bytes identity_ids = 1;
   Proof proof = 2;
 }
+
+message StateTransitionResultsRequest {
+  repeated bytes state_transition_hashes = 1;
+  bool prove = 2;
+}
+
+message StateTransitionResultsResponse {
+  oneof responses {
+    string errors = 1;
```

Comment:
As I remember error contains code, log (with json of data and something else). Please check how we handle errors from ABCI in DAPI

Reply:
We didn't handle them before, we ere just throwing a GrpcError with the stringified JSON from ABCI response

Reply:
I don't think so. We parse log and put data to GrpcError metadata as I remember.